Source/DafnyCore/Backends/CSharp/CsharpCodeGenerator.cs

```diff
@@ -187,7 +187,9 @@ protected override void EmitBuiltInDecls(SystemModuleManager systemModuleManager
     private void EmitFuncExtensions(SystemModuleManager systemModuleManager, ConcreteSyntaxTree wr) {
       var funcExtensions = wr.NewNamedBlock("internal static class FuncExtensions");
       wr.WriteLine("// end of class FuncExtensions");
-      foreach (var kv in systemModuleManager.ArrowTypeDecls) {
+      var arrowTypeDecls = systemModuleManager.ArrowTypeDecls.ToList();
+      arrowTypeDecls.Sort((pair, valuePair) => pair.Key - valuePair.Key);
```

Comment:
"pair"/"valuePair" really threw me off, as I thought this was deconstructing dictionary key/value pairs at first. How about just "left"/"right"?